src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -1502,9 +1502,15 @@ window.qBittorrent.DynamicTable ??= (() => {
             const rows = this.rows.getValues();
```

Comment:
One last comment, this line can be moved to just above the for loop at line 1515. To minimize redundant operations.